src/mcc-group.ts

```diff
@@ -0,0 +1,69 @@
+'use strict'
+
+import type {
+  Marqeta,
+  MarqetaOptions,
+  MarqetaError,
+} from './'
+
+export interface HoldIncrease {
+  type: string;
+  value: number;
+}
```

Comment:
This is the first time we have had an Object with `type` and `value`? It seems this would have been used many times already.

Reply:
First time i have seen it in the Marqeta api docs